src/Security/src/Authorization.Certificate/CertificateAuthorizationPolicies.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+namespace Steeltoe.Security.Authorization.Certificate;
+
+/// <summary>
+/// Use this class to refer to authorization policies for use with Cloud Foundry style certificates.
+/// <example>
+/// <code><![CDATA[
+/// [Authorize(Policy = CertificateAuthorizationPolicies.SameOrganization)]
+/// [HttpGet]
+/// public string AnyAppInTheOrgCanAccess()
+/// {
+///    return "Certificate is valid, client and server are in the same org.";
+/// }
+/// ]]>
+/// </code>
+/// </example>
+/// </summary>
+public static class CertificateAuthorizationPolicies
+{
+    /// <summary>
+    /// Restrict access to application instances operating within the same organization.
+    /// </summary>
+    public const string SameOrganization = "sameorg";
```

Comment:
Now that we only have strings exposed, is it (still?) possible to set a default policy?

Reply:
Hm.. yeah it's too streamlined now, looks a lot less easy than it should be for some of the options Microsoft exposes. Making the `IAuthorizationRequirement`s and `AuthorizationPolicyBuilderExtensions` public again would be a good idea

Reply:
Would it be hard to add a test for setting same-space/org as the default policy? I'm asking because `AuthorizationPolicyBuilder.RequireSameOrg/Space` _look_ like they shouldn't be public (because they are only used internally in library code).

The same applies to `SameOrgRequirement`/`SameSpaceRequirement`.

Background: Some time ago, we discussed and decided to delete an extension method in Discovery.Eureka that was never called, so I did. It was only later that I realized the removal broke the samples and we needed it anyway.

Reply:
The new test is nice!